src/rnn.jl

```diff
@@ -251,38 +259,40 @@ function rnnparam(r::RNN, w, layer::Integer, id::Integer, par::Integer; handle=c
             i2 = i1 + len - 1
         end
     end
+    @inline access(a, rng) = (isa(a, KnetArray) || ~useview) ? a[rng] : view(a, rng)
```

Comment:
I should implement view for KnetArray instead of forcing you to define a function here.  Of course only contiguous views will be supported and they will just call getindex, which is already implemented as view in KnetArray. I'll do this if we confirm that using views with Arrays really helps the speed (which may not, since the backward pass still needs to use separate arrays).

Reply:
I looked at this and it is difficult to do it in general.  The user will get confused about when view is supported and when it is not with KnetArrays.  However for this particular case, instead of defining a new function you can safely say:
```
Base.view(a::KnetArray,i::AbstractUnitRange)=a[i]
```

Reply:
In light of your latest comments, we could just get rid of views all together.  Did you try them with AutoGrad, and did they fail?

Reply:
I tried them 2 weeks ago while working on `batchnorm`. It was failed in gradchecks, so I did not push it. I don't remember the actual error log, however. Also, typing is a problem
```
julia> a = rand(10,10);

julia> b = view(a, 1:2,1:3)
2×3 SubArray{Float64,2,Array{Float64,2},Tuple{UnitRange{Int64},UnitRange{Int64}},false}:
 0.507469  0.979536  0.292401
 0.211171  0.509129  0.335866

julia> isa(b, Array{Float64})
false

julia> isa(a, Array{Float64})
true
```